test/random-battles/gen7.js

```diff
@@ -13,6 +13,62 @@ describe('[Gen 7] Random Battle', () => {
 		testNotBothMoves('chimecho', options, 'calmmind', 'yawn');
 	});
 
+	it('should give Azumarill Aqua Jet', () => {
+		testSet('azumarill', options, set => {
+			assert(set.moves.includes('aquajet'), `Azumarill: got ${set.moves}`);
+		});
+	});
+
+	it('should give Typhlosion Eruption', () => {
+		testSet('typhlosion', options, set => {
+			assert(set.moves.includes('eruption'), `Typhlosion: got ${set.moves}`);
+		});
+	});
+
+	it('should not generate Pursuit as the only Dark STAB move', () => {
+		testSet('weavile', options, set => {
```

Comment:
GitHub decided to be crappy and lose my half-written comment but this should be programmatically done.

Reply:
```ts
const dex = Dex.forFormat(options.format);
const darkTypesWithPursuit = dex.species
    .all()
    .filter(pkmn => pkmn.types.includes('Dark') && pkmn.randomBattleMoves.includes('pursuit'))
    .map(pkmn => pkmn.id);
for (const pokemon of darkTypesWithPursuit) {
    testSet(pokemon, options, ...);
}
```